ide/coqide/unicode_bindings.ml

```diff
@@ -48,9 +48,11 @@ let load_files filenames =
   let add f =
     selected_filenames := f::!selected_filenames in
   let warn_default_not_found () =
+    let env = Boot.Env.init () in
+    let coqlib = Boot.Env.coqlib env |> Boot.Path.to_string in
     Ideutils.warning (Printf.sprintf
       "Warning: the file 'ide/default.bindings' was not found in %s."
-      (Envars.coqlib())) in
+      coqlib) in
```

Comment:
This change means that if coqlib isn't found we `exit 1` instead of printing to console, I don't think that's a good idea.

Reply:
What do you propose? We could let callers of `Env.init` recover the error, but as far as I can see there is no situation where if the file env is not setup, things would work.

Note also that in this particular case, the call to `Envars.coqlib ()` could return indeed `""` which was even more confusing IMHO, as it would print the message "not found" but don't point out the real cause of the issue [missing call to `coqlib_guess` I think]

Reply:
I agree tho, we should improve this before merge.

Reply:
Putting needs:progress for this then

Reply:
See the latest commit, actually, the whole message was bogus to start with, IIUC.